src/components/AssetInfoModal.vue

```diff
@@ -35,8 +35,8 @@
             <p class="text-sm font-bold">{{ emissionAmount }}</p>
           </div>
           <div class="w-1/2">
-            <small class="uppercase text-gray-500">Decimal Places</small>
-            <p class="text-sm font-bold">{{ asset?.decimals ?? 0 }}</p>
+            <small class="uppercase text-gray-500">Balance</small>
+            <p class="text-sm font-bold">{{ confirmedBalance ?? 0 }}</p>
```

Comment:
Great work! Let's make use of `$filters.formatBigNumber()` to improve balance displaying.